services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -216,9 +236,43 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
         dynamic_sidecar_client = get_dynamic_sidecar_client(app)
         dynamic_sidecar_endpoint = scheduler_data.dynamic_sidecar.endpoint
 
-        logger.info("Calling into dynamic-sidecar to restore state")
-        await dynamic_sidecar_client.service_restore_state(dynamic_sidecar_endpoint)
-        logger.info("State restored by dynamic-sidecar")
+        # disable file system event watcher while writing
+        # the outputs directory to avoid data being pushed
+        # via nodeports upon change
+        await dynamic_sidecar_client.service_disable_dir_watcher(
```

Comment:
use a context manager to ensure the dir_watcher always gets set back whatever happen